src/js/structs/Service.js

```diff
@@ -81,12 +86,32 @@ module.exports = class Service extends Item {
     };
   }
 
+  getStatus() {
+    let {tasksRunning} = this.getTasksSummary();
+    let deployments = this.getDeployments();
+
+    if (deployments.length > 0) {
+      return ServiceStatus.DEPLOYING.displayName;
+    }
+
+    if (tasksRunning > 0) {
+      return ServiceStatus.RUNNING.displayName;
+    }
+
+    let instances = this.getInstances();
+    if (instances === 0) {
+      return ServiceStatus.SUSPENDED.displayName;
+    }
+  }
+
   getTasksSummary() {
     return {
       tasksHealthy: this.get('tasksHealthy'),
       tasksRunning: this.get('tasksRunning'),
       tasksStaged: this.get('tasksStaged'),
-      tasksUnhealthy: this.get('tasksUnhealthy')
+      tasksUnhealthy: this.get('tasksUnhealthy'),
+      tasksUnknown: this.get('tasksRunning') -
```

Comment:
Yes it is a number which is only aggregated by the other statuses but we will need it at multiple places where the status counts are used. I thought the best option was to implement it here instead of duplicating the code all the time. This state is used for applications which a) have no health checks defined, or b) did not become healthy yet.